### DISCUSSION TITLE
Visual regression testing for ladle stories

### DISCUSSION THREAD

**Discussion by d-ivashchuk** [discordjs/discord.js#9172] [Ideas]
Hey team! I was browsing ladle dependants and was excited to see Discord in the list. Love & use discord daily so thought it would be stupid not to try to offer you our visual regression testing solution **free of charge**. We are also open source https://github.com/lost-pixel/lost-pixel have very simple integration with Ladle(setup in couple of config files) and offer a platform for seamless visual regression testing flow https://lost-pixel.com/. 

If you think visual regression testing of your components is helpful please ping me and I will prepare a PR for you :D

**Comment by almostSouji** [comment]
Hi! From your OP I'm unsure if you think we are Discord (the company). If that's the case, we are not!
discord.js is an API wrapper for the Discord bot API and is not associated with the company at all.

**Reply by d-ivashchuk** [reply]
👋 hey @almostSouji! not gonna lie, at first thought that you are associated with Discord directly :D but this does not change my offer anyways, you are driving pretty big community project & are using Ladle, so in case you think you'd benefit from some visual regression tests I am all up for setting it up 🙌
